.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: GitHub CI
+
+on:
+  push:
+    paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
+      - '*.md'
+      - '*.yml'
+  pull_request:
+    paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
+      - '*.md'
+      - '*.yml'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout nxdk
+        uses: actions/checkout@v2
+        with:
+          repository: XboxDev/nxdk
+          submodules: recursive
+      - name: Set xbox_kernel_test_suite dir
+        id: test
+        run: echo "::set-output name=dir::projects/xbox_kernel_test_suite"
+      - name: Checkout xbox_kernel_test_suite
+        uses: actions/checkout@v2
+        with:
+          path: ${{ steps.test.outputs.dir }}
+      - name: Install dependencies
+        run: echo "/usr/lib/llvm-10/bin" >> $GITHUB_PATH
```

Comment:
Can you add a comment for the reason of need for this? Along with the side disclaimer of the number need an update over time.
Also, include `https://github.com/XboxDev/nxdk/issues/196` in the comment as a reference for known issue. Which apparently is not resolved.

Reply:
See https://github.com/XboxDev/nxdk/issues/196#issuecomment-812238967